cmake/projects/OpenCV/hunter.cmake

```diff
@@ -11,7 +11,30 @@ include(hunter_cmake_args)
 include(hunter_download)
 include(hunter_pick_scheme)
 
+
 # List of versions here...
+hunter_add_version(
+    PACKAGE_NAME
+    OpenCV
+    VERSION                                                     "3.4.3-p1"
+    URL
+    "https://github.com/hunter-packages/opencv/archive/v3.4.
+3-p1.tar.gz"
+    SHA1
+    3589a7f19fdaef200091231c4255c5e5bb70b217
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    OpenCV
+    VERSION
+    "3.4.3-p0"
```

Comment:
If this version is not working properly then there is no need to keep it. Since it's not in master yet, it's okay to do a remove.

Reply:
Yes, I'll have to drop that commit as it still fails despite working locally.

-- OpenCV ARCH: x86
-- OpenCV RUNTIME: vc14
-- OpenCV STATIC: OFF